packages/block-library/src/navigation/edit/inner-blocks.js

```diff
@@ -39,7 +40,6 @@ const LAYOUT = {
 export default function NavigationInnerBlocks( {
 	isVisible,
 	clientId,
-	appender: CustomAppender,
```

Comment:
I'm not sure if this is still being used by the navigation screen; might be worth preserving the option of passing in a custom appender.

Reply:
I'm not sure how to best address this feedback item, but feel free to push to the branch if you like.

Reply:
I played around with it and realised the previous code wasn't working on the nav editor (probably due to the changes we made to the block late last year) so I just removed the prop being passed to NavigationInnerBlocks altogether.